mlflow/utils/request_utils.py

```diff
@@ -43,7 +43,16 @@ def get_backoff_time(self):
         backoff_value = super().get_backoff_time()
         if self.backoff_jitter != 0.0:
             backoff_value += random.random() * self.backoff_jitter
-        return float(max(0, min(Retry.DEFAULT_BACKOFF_MAX, backoff_value)))
+        # The attribute `BACKOFF_MAX` was renamed to `DEFAULT_BACKOFF_MAX` in this commit:
+        # https://github.com/urllib3/urllib3/commit/f69b1c89f885a74429cabdee2673e030b35979f0
+        # which was part of the major release of 2.0 for urllib3
+        default_backoff = (
+            Retry.BACKOFF_MAX
+            if Version(urllib3.__version__) < Version("2.0")
+            else Retry.DEFAULT_BACKOFF_MAX
+        )
```

Comment:
if urllib3_version < Version("2.0"):
        retry = JitteredRetry(**retry_kwargs)
    else:
        retry = Retry(**retry_kwargs)

Reply:
Wondering why we hit this issue. `Retry.DEFAULT_BACKOFF_MAX` is valid in urllib3 < 2.0.

Reply:
I'll remove the version check - thanks for pointing it out :) 
As for the Retry.DEFAULT_BACKOFF_MAX in versions prior to 2.0, I can't find its existence in their repo as of this parent to the commit referenced in this PR: https://github.com/urllib3/urllib3/blob/c96cf403fb4f24d414f40faf4691174e4c54ea0b/src/urllib3/util/retry.py

Reply:
I'm also wondering where the version boundary is.